### ISSUE TITLE
Missing controller to invoke the underlying services?

### ISSUE THREAD

**Issue by gauravmahawar** [amit13081999/Assignment_Amit_walkover#1]
@amit13081999 can you please point me to the controllers which we will use to call the underlying services? I was unable to find them. Thanks.

**Comment by amit13081999** [comment]
@gauravmahawar  yes sir. src/main/java/com/pairlearning/exampleapi/resources/UserResource.java

**Comment by amit13081999** [comment]
sir the path to controller is
 src/main/java/com/pairlearning/exampleapi/resources/UserResource.java.
